amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseSerializer.java

```diff
@@ -193,10 +193,12 @@ public Lease fromDynamoRecord(Map<String, AttributeValue> dynamoRecord, Lease le
     }
 
     @Override
-    public Map<String, ExpectedAttributeValue> getDynamoExistantExpectation() {
+    public Map<String, ExpectedAttributeValue> getDynamoExistentExpectation(String leaseKey) {
```

Comment:
Can we add an integration test in DynamoDBLeaseRefresherIntegrationTest for the method updateLeaseWithMetaInfo ?

Reply:
added integration tests